projects/client/RabbitMQ.Client/src/client/impl/TcpClientAdapter.cs

```diff
@@ -41,8 +41,16 @@ public TcpClientAdapter(Socket socket)
 
         public virtual void Close()
         {
-            if(sock != null)
+            // TODO: Socket.Close() would be a better option here. MK.
```

Comment:
I didn't check the source. The docs didn't make me believe they are identical. Will remove this comment then.